src/App.tsx

```diff
@@ -1,14 +1,49 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { wait } from './utils/fetchClient';
+import { Status } from './types/Status';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+
+function hasError(message: string, setErrorMessage: (value: string) => void) {
```

Comment:
```suggestion
function handleError(message: string, setErrorMessage: (value: string) => void) {
```
And you can place it in your app so you don't need to pass `setErrorMessage` every time